src/worker.py

```
@@ -1329,6 +1314,331 @@ async def serve_static(path: str, env):
+        if not ws:
+            return
+        try:# sessions: session_id -> {ws, participant_id, display_name, position, direction, is_moving, seat_id}
+            ws.serializeAttachment(json.dumps({
+                "session_id":     session_id,
+                "participant_id": info["participant_id"],
+                "display_name":   info["display_name"],
+                "position":       info["position"],
+                "direction":      info["direction"],
+                "seat_id":        info.get("seat_id", ""),
+            }))
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)